components/director/internal/domain/webhook/repository.go

```diff
@@ -146,7 +162,7 @@ func (r *repository) Update(ctx context.Context, item *model.Webhook) error {
 	if err != nil {
 		return errors.Wrap(err, "while converting model to entity")
 	}
-	return r.updater.UpdateSingle(ctx, entity)
+	return r.updaterGlobal.UpdateSingleGlobal(ctx, entity)
```

Comment:
It looks to me that now i can get a client cert for app A in tenant 1 and update a webhook via the graphql api for app B in tenant 2 (security issue)

Reply:
Fixed.